CarboPlanner/.env

```
@@ -1,5 +1,5 @@
 // .env file created based on this article : https://github.com/lugg/react-native-config
 
 EXPO_PUBLIC_API_URL = https://yolov5-flaskapi-5qhj5kt2ta-lz.a.run.app/v1/object-detection
-EXPO_PUBLIC_UUID    = 80b109e0-ceb4-44bf-8e4e-ebfe1b8587a3
+EXPO_PUBLIC_UUID    = 4fd0772c-b478-4715-9d1b-627458fe77a0
```

```suggestion
EXPO_PUBLIC_UUID    = 80b109e0-ceb4-44bf-8e4e-ebfe1b8587a3
``` 

Please revert to the old database uuid